### DISCUSSION TITLE
Broken link on https://netbeans.apache.org/kb/docs/java/javase-deploy.html

### DISCUSSION THREAD

**Discussion by sebbASF** [apache/netbeans#6207] [General]
### Apache NetBeans version

Apache NetBeans 18

### What happened

See https://issues.apache.org/jira/browse/ASFSITE-18

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

https://issues.apache.org/jira/browse/ASFSITE-18

### JDK

https://issues.apache.org/jira/browse/ASFSITE-18

### Apache NetBeans packaging

Other

### Are you willing to submit a pull request?

No

**Comment by neilcsmith-net** [comment]
It's possibly at https://web.archive.org/web/20140123003605/https://netbeans.org/project_downloads/samples/Samples/Java/DeploymentTutorial.zip

Moved to discussions (as we have with other website reports) while we work out where to manage them. Thanks!
